moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -1,31 +1,43 @@
 <?xml version="1.0"?>
 <launch>
+
+  <!-- Gazebo specific options -->
+  <arg name="world_name" default="worlds/empty.world"/>
   <arg name="paused" default="false"/>
   <arg name="gazebo_gui" default="true"/>
-  <arg name="initial_joint_positions" doc="Initial joint configuration of the robot"
-       default="[GAZEBO_INITIAL_JOINTS]"/>
 
-  <!-- startup simulated world -->
-  <include file="$(find gazebo_ros)/launch/empty_world.launch">
-    <arg name="world_name" default="worlds/empty.world"/>
+  <!-- Robot spawn pose -->
+  <arg name="x" default="0"/>
+  <arg name="y" default="0"/>
+  <arg name="z" default="0"/>
+  <arg name="R" default="0"/>
+  <arg name="P" default="0"/>
+  <arg name="Y" default="0"/>
```

Comment:
I still think that providing these arguments individually is overkill. One can easily specify the `world_pose`.

Reply:
But it is more intuitive to specify
<arg name="z" value="1"/>
than
<arg name="world_pose" value="-z 1" />
and the second option can still be used as it is only a default value defined for world_pose in gazebo.launch.

Reply:
Usually, you will not specify a single component only, but all position/orientation components. Thus, I consider the argument `world_pose` and `world_pose:="-x 0 -y 0 -z 1"` much more comprehensible than just `z:=0`. But this is obviously very subjective. Let's see what other maintainers think about that.